docs/cases/api/cases-api/cases-api-find-cases-by-alert.asciidoc

```diff
@@ -1,33 +1,33 @@
 [[cases-api-find-cases-by-alert]]
-=== Find Cases by Alert
+=== Find cases by alert
 
-Retrieves all case IDs that have a specific alert attached to the cases.
+Uses an alert ID (`alert_id`) to retrieve case IDs of the cases the alert is attached to.
 
-NOTE: Console supports only Elasticsearch APIs. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to {kibana-ref}/console-kibana.html[Run {es} API requests].
+TIP: The case ID is identical to the case's saved object ID.
 
-NOTE: The case ID refers to the cases saved object ID field.
+NOTE: Console supports only Elasticsearch APIs. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to {kibana-ref}/console-kibana.html[Run {es} API requests].
 
 ==== Request URL
 
-`GET <kibana host>:<port>/api/cases/alerts/<alert ID>`
+`GET <kibana host>:<port>/api/cases/alerts/<alert_id>`
 
 ===== URL parts
 
-The URL must include the `alert ID` of the alert that you are interested in find the associated cases it is attached to.
+The URL must include the ID of the alert that is attached to the case.
```

Comment:
We might want to keep the specific field name (`alert ID`) here. It seems quicker to understand at a glance.

Reply:
@joepeeples as in - The URL must include the `alert ID` attached to the case.

Reply:
Almost -- but the alert is attached to the case, not the alert ID. Maybe more like the original -- it's a little wordier but accurate:
```suggestion
The URL must include the `alert ID` of the alert attached to the case
```